examples/engine/custom_server.py

```diff
@@ -0,0 +1,55 @@
+from sanic import Sanic, text
```

Comment:
Let us still keep it under `examples/runtime`, because we still want the two top levels be "frontend language" and "backend runtime". If you want to add a README.md, we can create a `examples/runtime/engine` folder.

Since now we have many different entry points (native generation api, openai api, engine, and frontend language), later we can organize our documentation website better.

Reply:
done. thanks